litestar/exceptions/http_exceptions.py

```diff
@@ -43,18 +44,18 @@ class HTTPException(LitestarException):
     """Exception status code."""
     detail: str
     """Exception details or message."""
-    headers: dict[str, str] | None
+    headers: dict[str, str] | None = None
     """Headers to attach to the response."""
-    extra: dict[str, Any] | list[Any] | None
+    extra: dict[str, Any] | list[Any] | None = None
     """An extra mapping to attach to the exception."""
 
     def __init__(
         self,
         *args: Any,
         detail: str = "",
         status_code: int | None = None,
-        headers: dict[str, str] | None = None,
-        extra: dict[str, Any] | list[Any] | None = None,
+        headers: dict[str, str] | None | EmptyType = Empty,
+        extra: dict[str, Any] | list[Any] | None | EmptyType = Empty,
```

Comment:
This behaviour should be documented in the args docstring

Reply:
added!